rules/brainstorming-idea-champion/rule.md

```diff
@@ -0,0 +1,38 @@
+---
+type: rule
+title: Do you have a brainstorming idea champion?
+uri: brainstorming-idea-champion
+authors:
+  - title: Seth Daily
+    url: https://www.ssw.com.au/people/seth-daily
+  - title: Piers Sinclair
+    url: https://www.ssw.com.au/people/piers-sinclair
+created: 2024-04-02T13:42:55.753Z
+guid: 0034f3e2-a3b5-4527-8r3d-45b27f2e4a1f
+---
+
+During a Brainstorming day, many great ideas are generated, discussed, and refined. But what happens to these ideas once the day is over? Without a dedicated individual to drive an idea forward, it risks being forgotten or losing momentum. This is where the Idea Champion comes in.
+
+<!--endintro-->
+
+The Idea Champion should be someone who believes in the idea's potential.
+
+## The Idea Champion's responsibilities
+
+* **Take accountability** for the idea from the end of the brainstorming day until a Scrum Team has been assigned and development has started
+* **Discuss** the idea and answer people's questions
+* **Communicate** the status of the idea in its [GitHub Discussion](/brainstorming-idea-farming)
+* **Push the idea forwards** and get closure (approval/rejection)
+
+### Where it begins
+
+* An Idea Champion should be assigned to each idea for a specific Brainstorming Day. This should be done when [allocating teams](/brainstorming-team-allocation)
```

Comment:
Will the idea champion replace the 'Team Lead' role mentioned in https://ssw.com.au/rules/brainstorming-team-allocation/#team-allocation
(the guy in change of the brainstorming team)

If yes we should update that rule as well (update term and link to this one)

Reply:
No it will not replace